rules/Php70/EregToPcreTransformer.php

```diff
@@ -281,7 +281,7 @@ private function _ere2pcre_escape(string $content): string
     /**
      * @param mixed[] $r
      */
-    private function processCurlyBracket(string $s, int $i, array &$r, int $rr): int
+    private function processCurlyBracket(string $s, int $i, array & $r, int $rr): int
```

Comment:
How is this happening?
Should be untouched.

Reply:
this is ECS run on php 8.1, not by rector

Reply:
the ECS fix diff shows the following on php 8.1:

```diff
     ---------- begin diff ----------
@@ -206,7 +206,7 @@
     /**
      * @param mixed[] $r
      */
-    private function processBracket(string $content, int $i, int $l, array &$r, int $rr): int
+    private function processBracket(string $content, int $i, int $l, array & $r, int $rr): int
     {
         // special case
         if ($i + 1 < $l && $content[$i + 1] === ')') {
@@ -281,7 +281,7 @@
     /**
      * @param mixed[] $r
      */
-    private function processCurlyBracket(string $s, int $i, array &$r, int $rr): int
+    private function processCurlyBracket(string $s, int $i, array & $r, int $rr): int
     {
         $ii = strpos($s, '}', $i);
         if ($ii === false) {
    ----------- end diff -----------


Applied checkers:

 * PhpCsFixer\Fixer\FunctionNotation\FunctionTypehintSpaceFixer
 * PhpCsFixer\Fixer\Operator\ConcatSpaceFixer
 * PhpCsFixer\Fixer\Operator\NotOperatorWithSuccessorSpaceFixer
 * PhpCsFixer\Fixer\Operator\UnaryOperatorSpacesFixer
 *
```

Reply:
The responsible rule should be ignored in ecs.php

Reply:
I will check in separate PR.